bentoml/cli/deployment.py

```diff
@@ -213,3 +227,240 @@ def check_deployment_status(archive_path, platform, region, stage, api_name):
         deployment.check_status()
 
     return cli
+
+
+def parse_key_value_string(key_value_string):
+    if key_value_string:
+        result = {}
+        for item in key_value_string.split(','):
+            if item is not None:
+                splits = item.split('=')
+                result[splits[0]] = splits[1]
+        return result
+    else:
+        return None
+
+
+def get_deployment_operator_type(platform):
+    return DeploymentOperator.Value(platform.upper())
+
+
+def display_response_status_error(status):
+    _echo('Error message: {}'.format(status.error_message), CLI_COLOR_ERROR)
+
+
+def display_deployment_info(deployment, output):
+    _echo('Deployment {} info\n\n'.format(deployment.name))
```

Comment:
remove this line? - most of time when people use json/yaml output, they may pipe the output to other CLI program